package/kubectl-shell.sh

```diff
@@ -14,8 +15,13 @@ if [ -z "${cluster}" ]; then
     exit 1
 fi
 
+if [ -z "${cacerts}" ]; then
+    echo No cacerts provided
```

Comment:
@moelsayed could you confirm it won't break for clusters on upgraded setups?

Reply:
Upgrading and downgrading rancher server will trigger a node agent update. I have tested upgrade from v2.1.6 to master and rollback to v2.1.6 again with no issues.